docs/release-strategy.md

```diff
@@ -29,10 +29,16 @@ PRs and Issues associated with the next two milestones will be prioritized for r
 
 ## Git Branches and Tags
 
-Every `X.Y` release stream gets a new branch.
+Every `X.Y` release stream gets a new branch `release-X.Y`. The release branch is created with first beta, release candidate, or final `X.Y.0` release.
 
 Each release, `X.Y.Z`, exists as a tag named `vX.Y.Z`.
 
+Alpha releases are tagged as `vX.Y.Za1`, `vX.Y.Za2`, and so on. Alpha releases are created from the main branch, **not** from a `release-X.Y` branch.
+
+Beta releases are tagged as `vX.Y.Zb1`, `vX.Y.Zb2`, and so on from `release-X.Y` branch.
```

Comment:
I've always used a tag as the base of a branch, too. So, tag a beta, make the release branch from the same point, then future tags for that release series are on the branch and clearly after only that initial branching version and not intermingled with pre-releases of other versions on the main branch.